backend/logic/deploy/cdn/gcp.js

```diff
@@ -281,6 +288,29 @@ async function createUrlMap(client, backendBucketLink, name) {
   return await get({ client, url: result.data.targetLink })
 }
 
+/**
+ * Invalidate a urlMap cache
+ *
+ * @param client {object} REST client with Oauth
+ * @param urlMapName {string} name (resource ID) of the urlMap
+ * @param paths {Array<string>} paths to invalidate (starting with /)
+ */
+async function invalidateCache(client, urlMapName, paths) {
+  for (const pth of paths) {
+    await post({
```

Comment:
Maybe add a comment that we could optimize this in the future by making concurrent requests.